archlinuxcn/cambalache/PKGBUILD

```diff
@@ -0,0 +1,23 @@
+# Maintainer: Dušan Simić <dusan.simic1810@gmail.com>
+
+pkgname=cambalache
+pkgver=0.16.0
+pkgrel=2
+pkgdesc='A new RAD tool for Gtk 4 and 3'
+arch=(x86_64)
+url=https://gitlab.gnome.org/jpu/cambalache
+license=(LGPL2.1)
+depends=(gtk3 gtk4 webkit2gtk-4.1 webkitgtk-6.0 gtksourceview4 glib2 python-lxml python-gobject libadwaita libhandy gobject-introspection)
+makedepends=(meson)
```

Comment:
```
cambalache W: Referenced python module 'merengue.getLogger' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_label.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_application.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_notebook.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_controller.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_center_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_assistant.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_css_provider.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_placeholder.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_widget.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_webkit2/mrg_webkit_web_view.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_grid.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_window.py'])
cambalache W: Referenced python module 'merengue.mrg_gtk.MrgGtkBox' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_carousel.py'])
cambalache W: Referenced python module 'merengue.utils' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_list_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_hdy_search_bar.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_widget.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_selection.py'])
cambalache W: Referenced python module 'merengue.MrgController' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_widget.py'])
cambalache W: Referenced python module 'merengue.MrgApplication' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/merengue'])
cambalache W: Referenced python module 'merengue.MrgPlaceholder' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_paned.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_selection.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_container.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_notebook.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_center_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_stack.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_assistant.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_list_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_bin.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_grid.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_box.py'])
cambalache W: Referenced python module 'merengue.mrg_gtk.MrgGtkWindow' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_window.py'])
cambalache W: Referenced python module 'merengue.mrg_gtk' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_application.py'])
cambalache W: Referenced python module 'merengue.mrg_gtk.MrgGtkWidget' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_webkit2/mrg_webkit_web_view.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_container.py'])
cambalache W: Referenced python module 'merengue.mrg_gtk.MrgGtkBin' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_hdy_search_bar.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_bin.py'])
cambalache W: Dependency gobject-introspection-runtime detected and implicitly satisfied (python modules ['gi.repository.Gio', 'gi.repository.GObject', 'gi.repository.GLib'] needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_paned.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_text_view.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_viewport.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_type_chooser_popover.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_revealer.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_hdy_search_bar.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_menu_item.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_project.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_property_label.py', 'usr/lib/python3.11/site-packages/cambalache/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_window.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_webkit2/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_label.py', 'usr/lib/python3.11/site-packages/cambalache/app/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_list_box_row.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_notebook.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_hdy_leaflet.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_list_box.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_icon_name_entry.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_overlay.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_webkit2/mrg_webkit_web_view.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_entry.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_ui_editor.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_spin_button.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_enum_combo_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_center_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_placeholder.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_layout_property.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_widget.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_object.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_window.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_grid.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_css_editor.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_db.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_translatable_widget.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_object_chooser.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_scrolled_window.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_create_editor.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_property.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_assistant.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_bin.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_hdy_deck.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_list_store.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_flags_entry.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_box.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_object_editor.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_fragment_editor.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_frame.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_aspect_frame.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_controller.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_css_provider.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_tutor.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_ui_requires_editor.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_popover.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_g_binding.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/merengue', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_menu.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_view.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_selection.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_window.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_toplevel_chooser.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_type_chooser.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_stack.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_expander.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_bin.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_application.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_context_menu.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_objects_base.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_signal_editor.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_file_entry.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_base.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_type_chooser_widget.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_controller_registry.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_child_type_combo_box.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_type_info.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_object_data_editor.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_application.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_color_entry.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_css.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_ui.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_carousel.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_text_buffer.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_source_view.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_application_window.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_object_data.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_switch.py'])
cambalache W: Dependency glibc detected and implicitly satisfied (libraries ['usr/lib/libc.so.6'] needed in files ['usr/lib/libcambalacheprivate-4.so', 'usr/lib/libcambalacheprivate-3.so'])
cambalache W: Dependency pango detected and implicitly satisfied (python modules ['gi.repository.Pango'] needed in files ['usr/lib/python3.11/site-packages/cambalache/app/cmb_window.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_icon_name_entry.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_signal_editor.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_tree_view.py'])
cambalache W: Dependency gcc-libs detected and implicitly satisfied (libraries ['usr/lib/libgcc_s.so.1'] needed in files ['usr/lib/libcambalacheprivate-4.so', 'usr/lib/libcambalacheprivate-3.so'])
cambalache W: Dependency python detected and implicitly satisfied (python modules ['logging', 'traceback', 'sqlite3', 'time', 'enum.Enum', 'os', 'locale', 'inspect', 'ctypes', 'sys', 'builtins', 'collections.namedtuple', 'warnings', 'tempfile', 'json', 'ast', 'signal', 'importlib', 'ctypes.util.find_library', 'math', 'socket'] needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_application.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_context_menu.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_css_provider.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_project.py', 'usr/bin/cambalache', 'usr/lib/python3.11/site-packages/cambalache/cmb_signal_editor.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_tutor.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_file_entry.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/utils.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_window.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_controller_registry.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_db.py', 'usr/lib/python3.11/site-packages/cambalache/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/merengue', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_view.py', 'usr/lib/python3.11/site-packages/cambalache/app/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_application.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_create_editor.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_css.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_icon_name_entry.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_spin_button.py'], programs ['python3'] needed in scripts ['usr/bin/cambalache', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/merengue'])
cambalache W: Dependency dconf detected and implicitly satisfied (needed for glib schemas)
cambalache W: Dependency gdk-pixbuf2 detected and implicitly satisfied (python modules ['gi.repository.GdkPixbuf'] needed in files ['usr/lib/python3.11/site-packages/cambalache/control/cmb_icon_name_entry.py'])
cambalache W: Dependency hicolor-icon-theme detected and implicitly satisfied (needed for hicolor theme hierarchy)
cambalache W: Dependency included, but may not be needed ('webkitgtk-6.0')
cambalache W: Dependency included, but may not be needed ('gobject-introspection')
```

Reply:
according to [upstream's issue](https://gitlab.gnome.org/jpu/cambalache/-/issues/196)
add
```
prepare() {
	sed -i "s/compileall.compile_dir(sys.argv\[1\], force=True)/compileall.compile_dir(destdir, force=True)/" "$srcdir/$pkgname-$pkgver/postinstall.py"
}
```
and works well.

Reply:
1. The compilation error has been resolved.
2. The errors detected by Namcap are still not fixed.
```
cambalache W: Referenced python module 'merengue.mrg_gtk.MrgGtkBin' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_bin.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_hdy_search_bar.py'])
cambalache W: Referenced python module 'merengue.mrg_gtk.MrgGtkWidget' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_container.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_webkit2/mrg_webkit_web_view.py'])
cambalache W: Referenced python module 'merengue.MrgController' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_widget.py'])
cambalache W: Referenced python module 'merengue.utils' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_list_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_widget.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_hdy_search_bar.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_selection.py'])
cambalache W: Referenced python module 'merengue.mrg_gtk.MrgGtkWindow' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_window.py'])
cambalache W: Referenced python module 'merengue.mrg_gtk' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_application.py'])
cambalache W: Referenced python module 'merengue.getLogger' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_widget.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_application.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_grid.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_center_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_controller.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_label.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_css_provider.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_webkit2/mrg_webkit_web_view.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_notebook.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_placeholder.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_assistant.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_window.py'])
cambalache W: Referenced python module 'merengue.MrgPlaceholder' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_handy/mrg_container.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_paned.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_selection.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_grid.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_bin.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_center_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_stack.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_list_box.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_notebook.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_gtk/mrg_gtk_assistant.py'])
cambalache W: Referenced python module 'merengue.MrgApplication' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/merengue'])
cambalache W: Referenced python module 'merengue.mrg_gtk.MrgGtkBox' is an uninstalled dependency (needed in files ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_adw/mrg_adw_carousel.py'])
cambalache W: Dependency glibc detected and implicitly satisfied (libraries ['usr/lib/libc.so.6'] needed in files ['usr/lib/libcambalacheprivate-4.so', 'usr/lib/libcambalacheprivate-3.so'])
cambalache W: Dependency glib2 detected and implicitly satisfied (libraries ['usr/lib/libgobject-2.0.so.0', 'usr/lib/libglib-2.0.so.0'] needed in files ['usr/lib/libcambalacheprivate-4.so', 'usr/lib/libcambalacheprivate-3.so'])
cambalache W: Dependency python detected and implicitly satisfied (python modules ['sys', 'json', 'inspect', 'ctypes.util.find_library', 'collections.namedtuple', 'warnings', 'os', 'traceback', 'socket', 'math', 'enum.Enum', 'signal', 'ast', 'logging', 'sqlite3', 'importlib', 'time', 'ctypes', 'tempfile', 'locale', 'builtins'] needed in files ['usr/bin/cambalache', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/utils.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_controller_registry.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_signal_editor.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_view.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_project.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_css_provider.py', 'usr/lib/python3.11/site-packages/cambalache/app/__init__.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_application.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_db.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_window.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/mrg_application.py', 'usr/lib/python3.11/site-packages/cambalache/priv/merengue/merengue', 'usr/lib/python3.11/site-packages/cambalache/cmb_context_menu.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_create_editor.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_icon_name_entry.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_spin_button.py', 'usr/lib/python3.11/site-packages/cambalache/control/cmb_file_entry.py', 'usr/lib/python3.11/site-packages/cambalache/cmb_css.py', 'usr/lib/python3.11/site-packages/cambalache/app/cmb_tutor.py', 'usr/lib/python3.11/site-packages/cambalache/__init__.py'], programs ['python3'] needed in scripts ['usr/lib/python3.11/site-packages/cambalache/priv/merengue/merengue', 'usr/bin/cambalache'])
cambalache W: Dependency gcc-libs detected and implicitly satisfied (libraries ['usr/lib/libgcc_s.so.1'] needed in files ['usr/lib/libcambalacheprivate-4.so', 'usr/lib/libcambalacheprivate-3.so'])
cambalache W: Dependency included, but may not be needed ('webkitgtk-6.0')
cambalache W: Dependency included, but may not be needed ('gobject-introspection')
```